AntWars/AI.cs

```diff
@@ -0,0 +1,75 @@
+﻿using AntWars.AI;
```

Comment:
Stimmt schon, ich dachte mir nur so kann man ganz gut erklären, an welcher Stelle die beiden AIs ins Spiel kommen.
Sonst haben wir nur die beiden abstrakten Klassen AIBase und AIAntBase, die nirgends abgeleitet werden, was irgendwie doof aussieht in einem Diagramm.